components/style/properties/longhand/inherited_text.mako.rs

```diff
@@ -750,6 +751,17 @@ ${helpers.single_keyword("text-align-last",
         pub color: Option<specified::CSSColor>,
     }
 
+    impl From<Shadow> for SpecifiedTextShadow {
+        fn from(shadow: Shadow) -> SpecifiedTextShadow {
```

Comment:
Yes, also `Shadow` has 2 additional fields but they are not getting parsed if `disable_spread_and_inset` argument is true in parse method(which is true in that property).